### PR TITLE
MAT-6175 get human readable for export

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-6175](https://jira.cms.gov/browse/MAT-6175)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/services/TranslationServiceClient.java [hunk 1]
```diff
@@ -41,4 +42,26 @@ public List<TranslatedLibrary> getTranslatedLibraries(String cql, String accessT
       throw new TranslationServiceException(msg, ex);
     }
   }
+
+  public String getMeasureBundleExport(Measure measure, String accessToken) {
```
Comment (adongare) [inline src/main/java/gov/cms/madie/services/TranslationServiceClient.java]: method name is a bit misleading. could be `getHumanReadable`?

Comment (sb-cecilialiu) [reply src/main/java/gov/cms/madie/services/TranslationServiceClient.java]: @adongare Just changed the name, thanks so much Ashok!

Comment (adongare) [reply src/main/java/gov/cms/madie/services/TranslationServiceClient.java]: Sure, np

**Review by adongare — APPROVED** [review]
